README.md

```diff
@@ -1,24 +1,173 @@
+          </a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/"><img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/postgresql/postgresql-plain-wordmark.svg" alt="PostgreSQL" width="55" height="55" /></a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **User Registration and Authentication**
+- **User Profiles**
+- **Blog Post Management**
+- **Commenting and Interaction**
+- **API for External Access**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any web-browser.
+
+### Setup
+
+Clone this [repository](https://github.com/lorenzoworx/velvetScribe.git) to your desired folder.
+
+### Install
+
+This project does not require installation.
+
+### Usage
+
+**To run the project follow the following instruction:-**
+
+- Clone this [repository](https://github.com/lorenzoworx/velvetScribe.git) in your local machine with correct branch.
+- Run following cmds:-
+- **rails db:create**
+- **rails db:migrate**
+- **rails s**
+
+### Run tests
+
+To run tests, run the following command: **rspec**.
+
+### Deployment
+
+You can deploy this project using: **Not available**.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Okolo Oshoke**
+
+- GitHub: [@lorenzoworx](https://github.com/lorenzoworx)
+- Twitter: [@lorenzoworx](https://twitter.com/lorenzoworx)
+- LinkedIn: [Oshoke Okolo](https://www.linkedin.com/in/oshokeokolo/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Validations and Model Specs**
+- [ ] **Processing Data in Models**
+- [ ] **Setup and Controllers**
+- [ ] **Controllers Specs**
+- [ ] **Views**
+- [ ] **Forms**
+- [ ] **Integration Specs for Views and Fixing N+1 Problems**
+- [ ] **Add Devise**
+- [ ] **Add API Endpoints**
+- [ ] **API Documentation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, be sure to leave a 🌟. Your support is greatly appreciated!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the microverse community for challenging me to build this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] I would kindly suggest you work on the MIT license link as the link you have there currently is broken and throwing error 404. Having a working link in your readme me file makes it look professional.

MIT 👇 

![mit](https://github.com/lorenzoworx/velvetScribe/assets/78634841/856cf132-6c48-4e42-8894-1fe3f93e5aa7)